### PR TITLE
Setting property PO

### PR DESCRIPTION
**Pull request by Saralrat**
From : http://redmine.ecosoft.co.th:10083/issues/5960

1. On the Purchase Invoice page, always show the Supplier Invoice section.

Screenshot from 2024-12-11 10-50-26

2. When you fill in the Supplier Invoice section, it will automatically pull you to the Tax Invoice tab.

Screenshot from 2024-12-11 10-50-46

### CODE REVIEW

File: menghua_co/hooks.py [hunk 1]
```diff
@@ -252,6 +252,7 @@
 	"Sales Order": "public/js/sales_order.js",
```
Comment (newtratip) [inline menghua_co/hooks.py:252]: Please move space of this line same with other lines.

File: menghua_co/fixtures/property_setter.json [hunk 2]
```diff
@@ -197,5 +197,14 @@
...
   "value": "0"
+ },
+ {
+    "doc_type": "Purchase Invoice",
+    "doctype": "Property Setter",
+    "doctype_or_field": "DocField",
+    "field_name": "supplier_invoice_details",
+    "name": "Purchase Invoice-supplier_invoice_details-collapsible",
+    "property": "collapsible",
+    "property_type": "Check"
```
Comment (newtratip) [inline menghua_co/fixtures/property_setter.json:208]: This property setter from you input property to file manually or from export fixture by system

I see space first of line not same with other lines

File: menghua_co/hooks.py (hunk 1 above)
Comment (Saralrat) [reply menghua_co/hooks.py]: > Please move space of this line same with other lines.

Completely edited.

File: menghua_co/fixtures/property_setter.json (hunk 2 above)
Comment (Saralrat) [reply menghua_co/fixtures/property_setter.json]: Completely edited.

File: menghua_co/hooks.py (hunk 1 above)
Comment (Saralrat) [reply menghua_co/hooks.py]: Completely edited.

**Review by newtratip — APPROVED** [review]
LGTM 👍

**Pull request merged**